mission/config/functions.hpp

```diff
@@ -202,9 +202,12 @@ class CfgFunctions
 			class director_init {};
 			class director_job {};
 			class director_check_mission_end {};
+			class director_complete_zone {};
 			class director_open_connected_zones {};
 			class director_open_zone {};
+			// possibly BN specific?
 			class director_open_closest_zone {};
```

Comment:
Need to check whether we need to use closest zone, or if we can go back to using connectors. I know a lot of the map editors have been ignoring connectors and this could cause problems for them.

Reply:
pushed changes.